src/App.js

```diff
@@ -74,11 +84,13 @@ class App extends Component {
           <Navigation dynamic="off-home-nav" />
           {!!this.state.mySongs.length && 
           <MySongLibrary
-          songs={ this.state.mySongs } 
+          songs={ this.state.mySongs }
           mySongs={this.state.mySongs }
-          handleSong={this.removeSong} 
+          handleSong={this.removeSong}
           buttonIcon={[<MdRemoveCircle className="handle-song-icon"/>]}
           />}
+          {this.state.error &&
+          <h2>{this.state.error}</h2>}
```

Comment:
we can add a class here for styling, but otherwise nice work on the conditional render